ansible/roles/debops.postfix/defaults/main.yml

```diff
@@ -256,6 +271,351 @@ postfix__tls_dh512_param_file: '{{ ansible_local.dhparam[postfix__dhparam_set]
+# When enabled, `postfix` will configure mail-boxes locally and allow LDAP authenticated users to
+# send email over this instance.
+postfix__ldap_enabled: 'False'
+                                                                   # ]]]
+# .. envvar:: postfix__virtual_mail_posix_user [[[
+#
+# Virtual Mail POSIX username
+# For the accesses to the mailbox directories a separate user `vmail` (`Virtual Mail`) is created,
+# under which the accesses of Postfix, Dovecot and other components of the mail server should take place.
+# On the one hand this prevents mail server components from accessing sensitive system directories,
+# on the other hand it protects the mailboxes from external access.
+# Only vmail (and root) are allowed to access the mailboxes.
+postfix__virtual_mail_posix_user: 'vmail'
+                                                                   # ]]]
+# .. envvar:: postfix__virtual_mail_posix_uidnumber [[[
+#
+# Virtual Mail POSIX uidNumber
+#
+postfix__virtual_mail_posix_uidnumber: 5000
+                                                                   # ]]]
+# .. envvar:: postfix__virtual_mail_posix_group [[[
+#
+# Virtual Mail POSIX group
+#
+postfix__virtual_mail_posix_group: 'vmail'
+                                                                   # ]]]
+# .. envvar:: postfix__virtual_mail_posix_gidnumber [[[
+#
+# Virtual Mail POSIX gidNumber
+#
+postfix__virtual_mail_posix_gidnumber: 5000
+                                                                   # ]]]
+# .. envvar:: postfix__virtual_mailbox_base [[[
+#
+# All mailboxes are stored directly in the file system of the server.
+#
+# This setting describes the prefix that the virtual delivery agent prepends to all pathname results from
+# `$virtual_mailbox_maps` table lookups.
+# The directories under this base contain the users MailDirs.
+# This directory must be created in advance and must be writeable by all potential mail users.
+# To achieve this, either use the same GID across all mail users and set group ownership to that GID,
+# or make the directory world writable.
+postfix__virtual_mailbox_base: '/var/vmail'
+                                                                   # ]]]
+# .. envvar:: postfix__virtual_alias_maps [[[
+#
+# Optional lookup tables that alias specific mail addresses or domains
+# to other local or remote address.
+# The default option, when using Virtual Mail with LDAP, will first go over the
+# alisaes set by :ref:`debops.etc_aliases` and if no match is found query
+# the LDAP for the email address.
+#
+# Specify zero or more `type:name` lookup tables, separated by whitespace or comma.
+# Tables will be searched in the specified order until a match is found.
+# Note: these lookups are recursive.
+postfix__virtual_alias_maps:
+  - '$alias_maps'
+  - 'ldap:/etc/postfix/ldap/virtual_aliases.cf'
+                                                                   # ]]]
+
+# LDAP authentication [[[
+# ^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
That's the wrong header marker for that particular file.

Reply:
Why?

Reply:
reStructuredText uses header markers to define the document structure, but they are expected to be the same on the same level. With this, you just put a few variables into a sub-section of the previous section. Sphinx will happily parse this, but that's not what you most likely want here.